### PR TITLE
🐛 In case RabbitMQ has unexpected error, wait before restarting it

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

Problem:
- If for any reason the RabbitMQ communication fails and the consumer in the webserver raises exception, it tries right away to restart and if for some reason this not work it enters a loop, thus loading the webserver.

Fix:
- add a sleep interval of 2 seconds before restarting

### CODE REVIEW

**Review by odeimaiz — APPROVED** [review]
The Man!

File: services/web/server/src/simcore_service_webserver/computation_subscribe.py [hunk 1]
```diff
@@ -114,6 +114,7 @@ async def events_message_parser(app: web.Application, data: bytes) -> None:
 
 
 APP_RABBITMQ_POOL_KEY = f"{__name__}.pool"
+_RABBITMQ_INTERVAL_BEFORE_RESTARTING_CONSUMER_S = 2
```
Comment (pcrespov) [inline services/web/server/src/simcore_service_webserver/computation_subscribe.py:117]: MINOR: add "WAIT_BEFORE" in the name analogous to tenacity naming

**Review by pcrespov — APPROVED** [review]

**Review by mrnicegyu11 — APPROVED** [review]
Yes its what we need 💯

File: services/web/server/src/simcore_service_webserver/computation_subscribe.py (hunk 1 above)
Comment (pcrespov) [reply services/web/server/src/simcore_service_webserver/computation_subscribe.py:117]: Can we conclude some best practices from here? e.g.

- Every restart should include a timeout
- This timeout can be fix or random, ... (as tenacity options) depending on the context
and

- Should we create an generic/abstract mechanism to enforce this?
- Should we review other parts of the code and apply consistently

Comment (sanderegg) [reply services/web/server/src/simcore_service_webserver/computation_subscribe.py:117]: we should use tenacity

Comment (sanderegg) [reply services/web/server/src/simcore_service_webserver/computation_subscribe.py:117]: with retry_unless_exception_type(CancelledError) | retry_unless_value(continue==False) and drop all this custom made code

**Pull request merged**